src/System.Runtime.Extensions/tests/Performance/Perf.ArrayList.cs

```diff
@@ -0,0 +1,454 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The ArrayList tests are all in System.Collections.NonGenerics\tests.  These should probably live there, too, until such time as they're all moved.

Reply:
I am moving it .....